### DISCUSSION TITLE
Google Analytics for the dashboard.

### DISCUSSION THREAD

**Discussion by priyabaljoseph** [statamic/cms#10641] [General]
I am happy to announce the launch of the plug-n (Ginsinsights) to seamlessly incorporate Google Analytics data into the Statamic CMS environment, offering users a comprehensive overview of website traffic directly within the dashboard as a widget and more extensive reports in a dedicated section.
Please try it out and let me know your feedback.
Vijaysoftware12/ginsights
Thanks,
PJ

**Comment by duncanmcclean** [comment]
Just a heads up: the `#show-and-tell` channel on Discord is more focused for announcing new addons. 😄

**Reply by priyabaljoseph** [reply]
Thank you, Duncan. I am going to post there also!
